ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketConfiguration.java

```diff
@@ -40,6 +40,7 @@
   private String authenticationCredentialsFile;
   private List<String> hostsAllowlist = Arrays.asList("localhost", "127.0.0.1");
   private File authenticationPublicKeyFile;
+  private String authenticationAlgorithm = "RS256";
```

Comment:
again can we use the JwtAlgorithm type for this

Reply:
86895b5